qiskit/providers/ibmq/api_v2/ibmqclient.py

```diff
@@ -25,17 +25,19 @@
 class IBMQClient:
     """Client for programmatic access to the IBM Q API."""
 
-    def __init__(self, api_token, auth_url):
+    def __init__(self, api_token, auth_url, proxies=None):
         """IBMQClient constructor.
 
         Args:
             api_token (str): IBM Q api token.
             auth_url (str): URL for the authentication service.
+            proxies (dict): proxies used in the connection.
         """
         self.api_token = api_token
         self.auth_url = auth_url
+        self.proxies = proxies
```

Comment:
Can we not store the attribute in the object, and instead pass it to `_init_service_clients()`? It would reflect better that is only relevant for the lower (session) layer.